contracts/smart-account/modules/SessionValidationModules/ISessionValidationModule.sol

```diff
@@ -24,4 +24,19 @@ abstract contract ISessionValidationModule {
         bytes calldata _sessionKeyData,
         bytes calldata _sessionKeySignature
     ) external virtual returns (bool);
+
+    /**
+     * @dev validates that the call (destinationContract, callValue, funcCallData)
+     * complies with the Session Key permissions represented by sessionKeyData
+     * @param destinationContract address of the contract to be called
+     * @param callValue value to be sent with the call
+     * @param funcCallData the data for the call. is parsed inside the SVM
```

Comment:
SVM is used first time here in the contract.. to make this abbreviation clear add in bracket `(SessionValidationModule)`

Reply:
fixed